lib/tools.js

```diff
@@ -89,8 +92,11 @@ function asJson (obj, num, time) {
 
   var value
   var notHasOwnProperty = obj.hasOwnProperty === undefined
-  if (serializers[wildcardGsym]) {
-    obj = serializers[wildcardGsym](obj)
+  if (formatters.log) {
+    obj = formatters.log(obj)
+  }
+  if (msg) {
```

Comment:
This is where the error is introduced. Since `msg === null` it is not getting logged as `"msg": null` (as #711 implements).